package.json

```diff
@@ -19,8 +19,7 @@
     "migrate:includes": "cp -R /Users/adam/neo4j/developer-resources/language-guides/_includes/ asciidoc/developer/modules/ROOT/pages/_includes",
     "migrate:menu": "node migrate/menu.js",
     "migrate:docs": "node migrate/docs.js",
-    "lint": "npm run lint-xref",
-    "lint-xref": "npm run lint-xref:unversioned && npm run lint-xref:labs-docs",
+    "lint": "npm run lint-xref:unversioned && npm run lint-xref:labs-docs",
```

Comment:
I think I finally understand what you meant. It was a bug, `npm run lint --prefix ui` was actually running the `npm run lint` task on the root (not on the ui directory).